cpmpy/solvers/ortools.py

```diff
@@ -480,9 +480,15 @@ def _post_constraint(self, cpm_expr, reifiable=False):
                 elif lhs.name == 'div':
                     return self.ort_model.AddDivisionEquality(ortrhs, *self.solver_vars(lhs.args))
                 elif lhs.name == 'element':
-                    # arr[idx]==rvar (arr=arg0,idx=arg1), ort: (idx,arr,target)
-                    return self.ort_model.AddElement(self.solver_var(lhs.args[1]),
-                                                     self.solver_vars(lhs.args[0]), ortrhs)
+                    arr, idx = lhs.args
+                    if is_int(idx) and (idx < 0 or idx >= len(arr)):
```

Comment:
So, OR-Tools is fine with constants as long as it is within bounds?

Reply:
yes, this is tested

Reply:
(although I'd be fine with just removing the 2nd condition for simplicity)

Reply:
Agreed, no strong opinion on keeping or removing it

Reply:
I'll go with simplicity